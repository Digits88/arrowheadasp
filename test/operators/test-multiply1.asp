Test Multiply operator:
<hr>

2.5 * 1.3 = <%= 2.5 * 1.3 %><br>
5 * 2 = <%= 5 * 2 %><br>
"5" * 2 = <%= "5" * 2 %><br>
5 * "2" = <%= 5 * "2" %><br>
"5" * "2" = <%= "5" * "2" %><br>
IsNull(Null * 2) = <%= IsNull(Null * 2) %><br>
IsNull(1 * Null) = <%= IsNull(1 * Null) %><br>
IsNull(Null * Null) = <%= IsNull(Null * Null) %><br>
Empty * 2 = <%= Empty * 2 %><br>
2 * Empty = <%= 2 * Empty %><br>
Empty * Empty = <%= Empty * Empty %><br>


