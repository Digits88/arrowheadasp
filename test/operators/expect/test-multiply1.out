Test Multiply operator:
<hr>

2.5 * 1.3 = 3.25<br>
5 * 2 = 10<br>
"5" * 2 = 10<br>
5 * "2" = 10<br>
"5" * "2" = 10<br>
IsNull(Null * 2) = True<br>
IsNull(1 * Null) = True<br>
IsNull(Null * Null) = True<br>
Empty * 2 = 0<br>
2 * Empty = 0<br>
Empty * Empty = 0<br>


